connector.go

```diff
@@ -32,7 +32,7 @@ type Connector interface {
 	EvalTyped(expr string, args interface{}, result interface{}) (err error)
 	ExecuteTyped(expr string, args interface{}, result interface{}) (SQLInfo, []ColumnMetaData, error)
 
-	SelectAsync(space, index interface{}, offset, limit, iterator uint32, key interface{}) *Future
+	SelectAsync(space, index interface{}, offset, limit uint32, iterator Iter, key interface{}) *Future
```

Comment:
What's about the offset?

Reply:
I'm sorry, but I don't get it. What's wrong with offset?

Reply:
Why `offset, limit uint32` and not `offset uint32, limit uint32` or `offset int32, limit uint32`?

Reply:
It seems to me that everything is fine here.

1. `offset, limit uint32` is a short form of `offset uint32, limit uin32`
2. offset is [MP_UINT](https://www.tarantool.io/en/doc/latest/dev_guide/internals/iproto/requests/#iproto-select).

Reply:
Sorry, I'm not well-versed in Go. I get it now, everything is fine then.